### DISCUSSION TITLE
export to local file

### DISCUSSION THREAD

**Discussion by ke352802081770314** [iSoron/uhabits#978] [Help & FAQ]
In Settings, when I choose "Export backup" or "Export to CSV", I get a (sorry not sure about the English so I have to give the localized wording - maybe "Share via") "Freigeben über: Android Beam, Bluetooth, E-Mail..."
What I expected was the ability to store it locally in Documents or some other suitable place.
How do I do that? 
(Import on the other hand doesn't ask me from which application to import but gives me a file chooser, so by symmetry export should write somewhere where import could read from.)

**Comment by alittlebitofit** [comment]
Its a year since its reported.

So any updates?

**Comment by adrian5** [comment]
So I looked at my phone and it appears tapping the backups buttons **silently** creates backups under `/Android/data/org.isoron.uhabits/files/`, even when one rejects/exits the "send to App" prompt. Not sure if this is behavior that can be relied on.

Edit: I see this behavior is also covered in the [FAQ](https://github.com/iSoron/uhabits/discussions/689). Not sure why it's silent though.

**Reply by alittlebitofit** [reply]
Ok so how do I access that storage? Because it's not just hidden but empty for me.

And how do I save to a custom location?

**Reply by adrian5** [reply]
Do you happen to be on Android 12? I think it's more strict regarding filesystem access. I'm using an Android 11 based distro, which might explain the difference. Either way, I think most of us agree that there should be an explicit "Save as…" feature, so we can export our backups into a common folder (such as `/Downloads/`) for further action.

**Comment by exanrauzer** [comment]
Maybe the case is that the data is saved as a file - it is not sewn into app. Since it is quite sensitive - the information off data storage could be more exposed in the app interface. The FAQ section needs a TOC as well.
